### DISCUSSION TITLE
A few questions

### DISCUSSION THREAD

**Discussion by LysanderAlcander** [iptv-org/community#68] [Q&A] [answered]
Hey guys, I'm really new to IPTV and first of all wanted to thank you all for the work done so far, it's really appreciated.
Second, I've came across with many, many questions when using lists and apps.

1. Do players (Like VLC Player) benefit from having a dedicated hardware video decoding, like a GPU?, I think most channels are encoding in .h264 if I'm right...
2. What's the most recommended player for Windows? I'm using VLC for now but its a bit laggy.
3. How can I tailor my own lists so I can choose which channels to include?, zapping through so many can get a bit time consuming.

Regards!

**Comment by BellezaEmporium** [comment, marked as answer]
>
>     1. Do players (Like VLC Player) benefit from having a dedicated hardware video decoding, like a GPU?, I think most channels are encoding in .h264 if I'm right...

VLC uses hardware decoding, if you give it the possibility. Please look on Tools-> Preferences and have a look on Input/Codecs. If hardware acceleration is set to "Automatic" you're good to go. If not, make the change automatically there.
Another player I know that has hardware decoding is MPC-HC. Like VLC, it is actively maintained too, and I think of it to be more robust than VLC in terms of video decoding.

>
>     2. What's the most recommended player for Windows? I'm using VLC for now but its a bit laggy.

VLC does the job pretty well normally. My preference would be MPC-HC.

>
>     3. How can I tailor my own lists so I can choose which channels to include?, zapping through so many can get a bit time consuming.

You probably can make a mix of playlists. Look on the README and download the playlists separately, and make a CRON job that downloads it periodically. Fusion the playlists by removing the #EXTM3U on the other files (do not remove it on the first file though, it'll serve as base to copy and paste the other channels afterwards).

Hope I've cleared your doubts.
